docs/README-dev.md

```diff
@@ -58,11 +58,16 @@ Because of the above, include paths in source should always feature the full pat
 Note that the recommended way of building the project is to use separate build directory, where CMake will generate all targets. In this way, you can keep a clean source directory. The following shell commands illustrate this usage:
 
 ```sh
-  cd <colobot_repository>
+  git clone https://github.com/colobot/colobot.git
+  cd colobot
+  git switch master  # or dev (see above)
```

Comment:
I'd change comment to something like `# if you intend to work with last release code`. 
The switch to `dev` branch being default branch was meant to make it easier to work on more up-to-date code and create pull request to this branch. Making everyone switch to master anyway defeats this purpose.

Reply:
You mean switch this around? Like this?
```
git switch dev  # or master (see above)
```

Reply:
Switching it around doesn't make sense, because when you clone repository without any other flags you're already on `dev`. The point is to make it clear this line is optional.

Your comment might suggest that this is mandatory to switch to `master` branch before making any work, while in most cases it shouldn't be required. Unless you're package maintainer, or you want to fix some serious bug ASAP.

Reply:
I see. Then I'll just delete the line. The option of switching to master is already implied in "Branch setup".